### PR TITLE
Update docs for base64 encoding of password and kill auto analyze

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8624, https://github.com/pingcap/docs-cn/pull/8699, 
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -440,7 +441,7 @@ min-available-ratio = 0.05
 | --tidb-port *port* | TiDB server port (default = 4000) | `tidb.port` |
 | --tidb-status *port* | TiDB status port (default = 10080) | `tidb.status-port` |
 | --tidb-user *user* | User name to connect to TiDB | `tidb.user` |
-| --tidb-password *password* | Password to connect to TiDB | `tidb.password` |
+| --tidb-password *password* | Password to connect to TiDB. The password can be either plaintext or Base64 encoded. | `tidb.password` |
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-configuration.md:444]: LGTM

File: statistics.md [hunk 2]
```diff
@@ -331,7 +331,33 @@ When the ratio of the number of modified rows to the total number of rows of `tb
...
+
+1. Execute the following SQL statement to get the address and task `ID` of the TiDB instance that is executing the background `ANALYZE` task:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT ci.instance as instance, cp.id as id FROM information_schema.cluster_info ci, information_schema.cluster_processlist cp WHERE ci.status_address = cp.instance and ci.type = 'tidb' and cp.info like 'analyze table %' and cp.user = '' and cp.host = '';
+    ```
+
+    If there is no result, it indicates that no `ANALYZE` task is executing in the background.
```
Comment (xuyifangreeneyes) [inline statistics.md:346]:
```suggestion
    If there is no result, it indicates that no `ANALYZE` task is running in the background.
```
It seems that _execute_ is intransitive verb?

File: statistics.md [hunk 3]
```diff
@@ -331,7 +331,33 @@ When the ratio of the number of modified rows to the total number of rows of `tb
...
 
-Before v5.0, when you execute a query, TiDB collects feedback with `feedback-probability` and updates the histogram and Count-Min Sketch based on the feedback. **Since v5.0, this feature is disabled by default, and it is not recommended to enable this feature.**
+Before TiDB v5.0, when you execute a query, TiDB collects feedback with `feedback-probability` and updates the histogram and Count-Min Sketch based on the feedback. **Since v5.0, this feature is disabled by default, and it is not recommended to enable this feature.**
+
+Since TiDB v6.0, TiDB supports using the `KILL` statement to terminate an `ANALYZE` task running in the background. If you find that an `ANALYZE` task running in the background consumes a lot of resources and affects your application, you can terminate the `ANALYZE` task by taking the following steps:
+
+1. Execute the following SQL statement to get the address and task `ID` of the TiDB instance that is executing the background `ANALYZE` task:
```
Comment (xuyifangreeneyes) [inline statistics.md:338]:
```suggestion
1. Execute the following SQL statement to get the TiDB instance address and  the `ID` of the background `ANALYZE` task:
```

File: statistics.md [hunk 4]
```diff
@@ -331,7 +331,33 @@ When the ratio of the number of modified rows to the total number of rows of `tb
...
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT ci.instance as instance, cp.id as id FROM information_schema.cluster_info ci, information_schema.cluster_processlist cp WHERE ci.status_address = cp.instance and ci.type = 'tidb' and cp.info like 'analyze table %' and cp.user = '' and cp.host = '';
+    ```
+
+    If there is no result, it indicates that no `ANALYZE` task is executing in the background.
+
+2. Use a client to connect to the TiDB instance that is executing the background `ANALYZE` task, and then execute the following `KILL` statement:
```
Comment (shichun-0415) [inline statistics.md:348]:
```suggestion
2. Use a client to connect to the TiDB instance that is performing the background `ANALYZE` task, and then execute the following `KILL` statement:
```

File: statistics.md [hunk 5]
```diff
@@ -331,7 +331,33 @@ When the ratio of the number of modified rows to the total number of rows of `tb
...
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    KILL TIDB ${id};
+    ```
+
+    `${id}` is the `ID` of the background `ANALYZE` task obtained from the query of the previous step.
```
Comment (shichun-0415) [inline statistics.md:356]:
```suggestion
    `${id}` is the `ID` of the background `ANALYZE` task obtained in the previous step.
```

File: statistics.md [hunk 6]
```diff
@@ -331,7 +331,33 @@ When the ratio of the number of modified rows to the total number of rows of `tb
...
+
+    > **Note:**
+    >
+    > The above `KILL` statement only works on the TiDB instance that is executing the background `ANALYZE` task, so you must use a client to connect to that TiDB instance first. If you use a client to connect to another TiDB instance, or if there is a proxy between the client and TiDB, the `KILL` statement cannot terminate the background `ANALYZE` task. For more information, see [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md).
```
Comment (shichun-0415) [inline statistics.md:360]:
```suggestion
    > The above `KILL` statement only works on a TiDB instance that is executing a background `ANALYZE` task. Therefore, you must use a client to connect to that TiDB instance first. If you use a client to connect to another TiDB instance, or if there is a proxy between the client and TiDB, the `KILL` statement cannot terminate the background `ANALYZE` task. For more information, see [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md).
```

File: statistics.md (hunk 2 above)
Comment (qiancai) [reply statistics.md]: No, "execute" is transitive. See https://www.merriam-webster.com/dictionary/execute. 
But your suggestion looks better. 👍

File: statistics.md [hunk 7]
```diff
@@ -331,7 +331,33 @@ When the ratio of the number of modified rows to the total number of rows of `tb
...
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT ci.instance as instance, cp.id as id FROM information_schema.cluster_info ci, information_schema.cluster_processlist cp WHERE ci.status_address = cp.instance and ci.type = 'tidb' and cp.info like 'analyze table %' and cp.user = '' and cp.host = '';
+    ```
+
+    If there is no result, no `ANALYZE` task is running in the background.
+
+2. Use a client to connect to the TiDB instance that is performing the background `ANALYZE` task, and then execute the following `KILL` statement:
```
Comment (qiancai) [inline statistics.md:348]:
```suggestion
2. Use a client to connect to the TiDB instance where the background `ANALYZE` task is running, and then execute the following `KILL` statement:
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by xuyifangreeneyes — APPROVED** [review]

**Pull request merged**